models/balancer/arbitrum/balancer_v2_arbitrum_liquidity.sql

```diff
@@ -67,14 +67,14 @@ WITH pool_labels AS (
             day,
             pool_id,
             token,
-            SUM(COALESCE(delta, 0)) AS delta
+            SUM(COALESCE(delta, CAST(0 as int256))) AS delta
```

Comment:
minor: preferred approach is `INT256 '0'` rather than `CAST()`

please note, this is only the case when using hardcoded values. it is best to use `CAST()` when it's a column instead.